### PR TITLE
Eoc survey UI change

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Add new API to expose survey state (dismissed, sent , error) so contoso can redirect to their own screen after survey ends, updated survey UI to be it's own page appearing before end call page, added default page after survey is dismissed/sent

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3564104

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/localization/locales/en-US/strings.json [hunk 1]
```diff
@@ -237,7 +237,7 @@
         "grantedDueToRoleChangeToPresenter": "You're a presenter. You can share content and facilitate the meeting."
       }
     },
-    "surveyQuestion": "How was your call today?",
+    "surveyQuestion": "Help us improve",
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/localization/locales/en-US/strings.json:240]: This is less a question now and more a `surveyTitle`

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/localization/locales/en-US/strings.json (hunk 1 above)
Comment (carocao-msft) [reply packages/react-composites/src/composites/localization/locales/en-US/strings.json]: Updated

File: packages/communication-react/review/beta/communication-react.api.md [hunk 2]
```diff
@@ -794,15 +794,16 @@ export interface CallCompositeStrings {
     startSpotlightParticipantListMenuLabel: string;
     stopSpotlightOnSelfParticipantListMenuLabel: string;
     stopSpotlightParticipantListMenuLabel: string;
-    surveyCancelButtonAriaLabel: string;
+    surveyCancelButtonLabel: string;
```
Comment (carocao-msft) [inline packages/communication-react/review/beta/communication-react.api.md:797]: surveySkipButtonLabel

File: packages/communication-react/review/beta/communication-react.api.md [hunk 3]
```diff
@@ -794,15 +794,16 @@ export interface CallCompositeStrings {
...
     surveyConfirmButtonLabel: string;
     surveyIssues: SurveyIssues;
     SurveyIssuesHeadingStrings: SurveyIssuesHeadingStrings;
-    surveyQuestion: string;
     surveyTextboxDefaultText: string;
+    surveyTitle: string;
     tagsSurveyHelperText: string;
     tagsSurveyQuestion: string;
     tagsSurveyTextFieldDefaultText: string;
+    thankYouForFeedBackString: string;
```
Comment (carocao-msft) [inline packages/communication-react/review/beta/communication-react.api.md:806]: endOfSurveyText

**Review by prabhjot-msft — APPROVED** [review]
Looks good, please address the suggested text changes before merging!

**Pull request merged**